backend/modules/pdf_processing/repository.py

```
@@ -1 +1,352 @@
+            return True
+            
+        except Exception as e:
+            debug_log(f"[REPOSITORY] Failed to save autofill data: {str(e)}")
+            self.db.rollback()
+            # Update scenario status to draft even on error
+            try:
+                scenario = self.db.query(Scenario).filter(Scenario.id == scenario_id).first()
+                if scenario:
+                    scenario.status = "draft"
+                    self.db.commit()
+            except:
```

Ruff (0.14.5)

Do not use bare `except`

(E722)